cores/esp32/esp32-hal-log.c

```diff
@@ -0,0 +1,42 @@
+#ifndef __MY_LOG__
+#define __MY_LOG__
+#include "stdio.h"
+#include "esp32-hal-log.h"
+#ifdef USE_ESP32_LOG
+    void log_to_esp(esp_log_level_t level, const char *format, ...)
```

Comment:
please pass also the TAG as argument:
```cpp
void log_to_esp(esp_log_level_t level, const char *tag, const char *format, ...)
```

Reply:
Here i wanted to replicate the actual behavior of the arduino library, so the tag is fixed.
But I understand it's better to put it as a parameter ( and passing it fixed when calling MACROs)